Add some missing entries to the changelog for -23 (#349)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

   -23

   * fixed percent-encoding of URI examples

>    * fixed an example that used a client hostname where a wallet one would be more appropriate

Author: It's not a very good sentence, sorry. Maybe this is better:

```suggestion
   * fixed an example that used 'client' where 'wallet' is more appropriate
```
